File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1050_basico.js
import { question } from "readline-sync";

function main() {
    const DDD = get_number();

    let destination;

    switch (DDD) {
        case 61:
            destination = `Brasilia`;
            break;
        case 71:
            destination = `Salvador`;
            break;
        case 11:
            destination = `Sao Paulo`;
            break;
        case 21:
            destination = `Rio de Janeiro`;
            break;
        case 32:
            destination = `Juiz de Fora`;
            break;
        case 19:
            destination = `Campinas`;
            break;
        case 27:
            destination = `Vitoria`;
            break;
        case 31:
            destination = `Belo Horizonte`;
            break;
        default:
            destination = `DDD nao cadastrado`;
            break;
    }
    print(`${destination}`);
}

function get_number() {
    return parseInt(question());
}

function print(texto) {
    return console.log(texto);
}

main();

File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1050_basico.py
lista = {
    '61': 'Brasilia',
    '71': 'Salvador',
    '11': 'Sao Paulo',
    '21': 'Rio de Janeiro',
    '32': 'Juiz de Fora',
    '19': 'Campinas',
    '27': 'Vitoria',
    '31': 'Belo Horizonte'
}

x = input()
# if x in lista:
#     print(lista[x])
# else:
#     print('DDD nao cadastrado')
print(lista.get(x, 'DDD nao cadastrado'))